src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -156,9 +156,12 @@ export const getIpIsNotNetworkOrBroadcastAddressSchema = (
     (value) => {
       const subnetAddr = getAddressObject(subnet, protocolVersion);
       if (subnetAddr) {
-        const subnetStart = subnetAddr?.startAddress().correctForm();
-        const subnetEnd = subnetAddr?.endAddress().correctForm();
-        return !(value === subnetStart || value === subnetEnd);
+        if (subnetAddr === null) return true;
+        else {
+          const subnetStart = subnetAddr?.startAddress().correctForm();
+          const subnetEnd = subnetAddr?.endAddress().correctForm();
+          return !(value === subnetStart || value === subnetEnd);
+        }
       }
       return true;
```

Comment:
```suggestion
        if (!subnetAddr) {
         return false;
        } else {
          const subnetStart = subnetAddr?.startAddress().correctForm();
          const subnetEnd = subnetAddr?.endAddress().correctForm();
          return !(value === subnetStart || value === subnetEnd);
        }
      }
```
@ammont82 any chance you can test this? ^

Reply:
I'm on it

Reply:
It works well!